packages/code-du-travail-frontend/server/server.js

```diff
@@ -70,6 +70,8 @@ nextApp.prepare().then(() => {
             "https://www.googletagmanager.com",
             "*.fabrique.social.gouv.fr",
             "https://cdnjs.cloudflare.com",
+            "https://cdn.jsdelivr.net",
+            "https://unpkg.com",
```

Comment:
we should not have those for the same reason we remove react-mathjax-preview
- https://cdnjs.cloudflare.com
- https://cdn.jsdelivr.net
- https://unpkg.com

Reply:
You right! I forgot to remove it because of use Direflow previously